e2e-tests/partials/insurance/memberOfAGroup.js

```diff
@@ -0,0 +1,11 @@
+import { INSURANCE_FIELD_IDS } from '../../constants/field-ids/insurance';
+
+const { MEMBER_OF_A_GROUP } = INSURANCE_FIELD_IDS.ELIGIBILITY;
+
+const memberOfAGroupPartial = {
+  summary: () => cy.get(`[data-cy="${MEMBER_OF_A_GROUP}-description"] summary`),
+  details: () => cy.get(`[data-cy="${MEMBER_OF_A_GROUP}-description"]`),
+  description: () => cy.get(`[data-cy="${MEMBER_OF_A_GROUP}-description-details"]`),
+};
+
+export default memberOfAGroupPartial;
```

Comment:
Should we remove `Partial` from the naming convention (and in the file above) since we don't do this with any other partials?